### DISCUSSION TITLE
Is it possible to style a table with rows having alternating background colours?

### DISCUSSION THREAD

**Discussion by sanga** [Textualize/rich#363] [Q&A] [answered]
Perhaps a picture is worth 1000 words: so as in https://datatables.net/examples/data_sources/dom.html

I find it much easier to visually "track" rows with that alternating background colour-scheme. My initial attempt to do this _kinda_ works but is not as beautiful as I might have hoped (i.e. the cells get alternating colours but the inter-cell spaces don't)
![2020-10-08-085044_grim](https://user-images.githubusercontent.com/2088/95420024-68c2fc80-0943-11eb-906c-64f86bce38a2.png)

The above pic was generated by this:
```python
bg = Style(bgcolor="rgb(40,40,40)")
table = Table(box=box.SIMPLE_HEAD, row_styles=["none", bg])
```

Oh and thanks for this awesome lib!

**Comment by willmcgugan** [comment, marked as answer]
Not currently, unless you were to set `box=None`

Its not quite what you are looking for, but you can set `row_styles=['dim', '']` which will display a similar affect.

I'll look in to extending the background color across the box lines in the next release.

**Reply by sanga** [reply]
Thanks. Would you rather I opened this as an issue or are you happy having it tracked here?

**Reply by willmcgugan** [reply]
An issue would be good to track it. 👍
